PubnativeLite/PubnativeLite/HyBid.h

```diff
@@ -69,6 +69,7 @@ FOUNDATION_EXPORT const unsigned char HyBidVersionString[];
 #import <HyBid/HyBidLogger.h>
 #import <HyBid/HyBidIntegrationType.h>
 #import <HyBid/HyBidAdSize.h>
+#import <HyBid/VWAdvertView.h>
 
```

Comment:
Maybe you need to add the imports for `VWAdRequest` and `VWContentCategory` to here as well?